docs/RFCS/range_repair.md

```diff
@@ -0,0 +1,94 @@
+- Feature Name: Range Repair
+- Status: draft
+- Start Date: 2015-08-19
+- RFC PR:
+- Cockroach Issue: #2149
+
+# Summary
+
+Add a new `Repair` service on each node that monitors all other nodes via
+gossip and determines if any of them are dead. When a node dies, the service
+looks through all the local ranges to see they have any replicas on the dead
+node. If any exist, remove the dead replica and add the local replica to the
```

Comment:
In practice, I think you'll want to queue up this work. Imagine a cluster with 10 nodes where one node dies temporarily. This would queue up 1/10th of the ranges for repair. Then the dead node comes back (e.g. a sysadmin kicked it or a machine reboot finished). You don't want to have to run through all of the affect ranges again.